### ISSUE TITLE
Transfer to GNOME infrastructure

### ISSUE THREAD

**Issue by phako** [intel/dleyna-core#55]
Hello,

I am currently maintaining a fork of all dLeyna projects in my personal workspace to push out fixes and improvements for dLeyna.

Parts of GNOME heavily depend on it. I propose that we transfer the whole project officially over to GNOME's infrastructure. You seem to have lost any interest, given that you even shut down the mailinglist without any prior notice:

![image](https://user-images.githubusercontent.com/55554/114023432-27b89000-9873-11eb-86ca-de51d3607c73.png)

**Comment by kanavin** [comment]
Hello, please go ahead, I fully support this. From what I can tell (from the outside), Intel's open source department is defunct, and there's simply no point in trying to revive this repo.

**Comment by debarshiray** [comment]
> I propose that we transfer the whole project officially over to
> GNOME's infrastructure.

Yes, please. At this point @phako 's fork is the de facto upstream, anyway.

**Comment by ebassi** [comment]
Would it be possible for @argretzi to archive this repository?
